Client/Classes/Handlers/ClientSquadHandler.sqx

```diff
@@ -70,5 +70,28 @@ namespace Intrusion.Client
 				};
 			};
 		};
+		
+		// Change unit uniform.
+		// _unit (Object): The unit to change dress on.
+		// _role (String): Role in vehicle.
+		// _vehicle (Object): The vehicle that the unit entered.
+		public static method SetUniform(
+			"_unit" as Object,
+			"_role" as String,
+			"_vehicle" as IntrusionVehicle)
+		{
+			private ["_professionType" as ProfessionType, "_uniformItems" as Array, "_uniformInfo" as UniformInfo];
+			_professionType = _vehicle.OwnerProfession;
+			_uniformItems = uniformItems _unit;
+			
+			_uniformInfo = [side _unit, _professionType, [_role] call VehicleRoleMeta.fromString] call gCommon.UniformConfig.GetUniformInfo;
+			_unit addUniform _uniformInfo.Uniform;
+			_unit addHeadgear _uniformInfo.Helmet;
+			
+			// Restore items/magazines etc to this units new uniform
```

Comment:
This is interesting. Does this mean that different uniforms can contain different number of items? In that case what will happen if all items do not fit, and maybe the uniform must be changed on server to...?